### PR TITLE
Add mobile page Javascript button actions 

### PR DESCRIPTION
**Pull request by benardop**
## changes implemented in this pull request
-  Add mobile page button click action to display the navigation links menu
-  Add button close action to close the navigation links

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,21 +9,23 @@
...
-                <ul class="navigation-bar">
-                    <li><a href="#" class="nav-link">Portfolio</a></li>
-                    <li><a href="#" class="nav-link">About</a></li>
-                    <li><a href="#" class="nav-link">Contact</a></li>
+                <ul id="navbar">
+                    <li><a href="javascript:void(0)" class="closebtn">&times;</a></li>
+                    <li><a href="#portfolio" class="nav-link">Portfolio</a></li>
+                    <li><a href="#about-me" class="nav-link">About</a></li>
+                    <li><a href="#contact" class="nav-link">Contact</a></li>
                 </ul>
```
Comment (Nemwel-Boniface) [inline index.html:26]: You have done a great job with your mobile portfolio menu. However, according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md), you are supposed to stick as much as possible to the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). You are currently left with the following styling changes to make your project perfect and ready for approval:

- [ ] Currently, your menu design does not match the chosen Figma design. When I tap the hamburger bars, it is supposed to cover the entire page of the screen as it is shown in the Figma design. The choice of colors you currently have also does not match the Figma design. You have a white background and it is supposed to be a bluish color as provided in the Figma design. Kindly fix this.

**This is what your design currently looks like:**

image

**This is the desired Figma design:**

image

- [ ] When I click on the hamburger bars, the bars as well as the logo are supposed to be hidden from view. Currently they are still visible. Kindly fix this.

**This is what your menu heading currently loos like:**

image

**This is the desired Figma design menu header:**

image

- [ ] Also, kindly note that the portfolio nav links are supposed to be placed at the left side of the screen. Kindly do so when you do the styling.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @benardop            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done a great job with your portfolio mobile version great job :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] Kindly note that according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md), you are supposed to add the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js). I have noticed that your project currently does not have Eslint included. Kindly include it.

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 2]
```diff
@@ -748,8 +801,22 @@ a {
...
+  height: 90%;
+  width: 100%;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background: #4756df;
+  overflow-y: hidden;
+  transition: 0.5s;
+  opacity: 0.96;
```
Comment (leonardodiasb) [inline css/styles.css:819]: - [ ] Your did a great job with the design of your mobile menu, it looks almost identical to the Figma template but on some screen it does not completely cover the entire screen. Kindly fix that.
Your design:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @benardop, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ]  As the previous Code Reviewer mentioned, you are supposed to add the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js). Your project currently does not have Eslint included. Kindly include it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @benardop 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**